### ISSUE TITLE
installing on iOS 13 Beta

### ISSUE THREAD

**Issue by matiasloda** [PopcornTimeTV/PopcornTimeTV#813]
App version: 3.2.35

Hi, there's anyone who tried to install it on apple tv with iOs 13 Beta?
Thanks

**Comment by Red-Mak** [comment]
Same question.
Thanks

**Comment by bweis** [comment]
Yes, it functions on the latest Beta.  Some UI Artifacts and images don't always load.  But that should be fixed with the next alamofire update.

**Comment by Red-Mak** [comment]
Thanks :)
